_maps/RandomZLevels/moonoutpost19.dmm

```diff
@@ -825,8 +825,7 @@
 	},
 /turf/open/floor/plating{
 	heat_capacity = 1e+006;
-	initial_gas_mix = "co2=48.7;n2=13.2;o2=32.4;TEMP=251";
-	temperature = 251
+	initial_gas_mix = "co2=48.7;n2=13.2;o2=32.4;TEMP=251"
```

Comment:
Can these var-edits be replaced with a variant object?

Reply:
In theory yes, but I both do not trust myself to deal with that alongside all the other stuff I'm balancing here and am afraid of how much shit like this there  is.

Reply:
Can you atomize the changes then?

Reply:
I'll consider it, I think I will go through and remove these when I'm done here, but I don't have the time for it as of now